<ui version="4.0" >
 <class>TransferHistory</class>
 <widget class="QWidget" name="TransferHistory" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>278</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <layout class="QHBoxLayout" >
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="clearButton" >
         <property name="text" >
          <string>Clear History</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="rangeType" >
         <item>
          <property name="text" >
           <string>Date</string>
          </property>
         </item>
         <item>
          <property name="text" >
           <string>Size</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="KTreeWidgetSearchLine" name="searchBar" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="clickMessage" >
          <string>Filter history...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionDelete_Selected" >
   <property name="text" >
    <string comment="delete selected transfer" >Delete Selected</string>
   </property>
  </action>
  <action name="actionDownload" >
   <property name="text" >
    <string>Download</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTreeWidgetSearchLine</class>
   <extends>KLineEdit</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
